test/run

```diff
@@ -279,11 +279,15 @@ test_from_dockerfile_minimal(){
   # ubi8/python-39-minimal / ubi8/python-39
   FULL_IMAGE_NAME=${IMAGE_NAME/-minimal/}
 
-  # mod_wsgi in micropipenv-requirements-test-app
-  # TODO: Find a better way how to specify full/minimal images that should work together
-  sed "s@#IMAGE_NAME#@${IMAGE_NAME}@;s@#FULL_IMAGE_NAME#@${FULL_IMAGE_NAME}@" $test_dir/from-dockerfile/mod_wsgi.Dockerfile.tpl > $test_dir/from-dockerfile/Dockerfile
-  ct_test_app_dockerfile $test_dir/from-dockerfile/Dockerfile $test_dir/micropipenv-requirements-test-app 'Hello World from mod_wsgi hosted WSGI application!' app-src
-  check_result $?
+  if ct_pull_image "$FULL_IMAGE_NAME"; then
+    # mod_wsgi in micropipenv-requirements-test-app
+    sed "s@#IMAGE_NAME#@${IMAGE_NAME}@;s@#FULL_IMAGE_NAME#@${FULL_IMAGE_NAME}@" $test_dir/from-dockerfile/mod_wsgi.Dockerfile.tpl > $test_dir/from-dockerfile/Dockerfile
+    ct_test_app_dockerfile $test_dir/from-dockerfile/Dockerfile $test_dir/micropipenv-requirements-test-app 'Hello World from mod_wsgi hosted WSGI application!' app-src
+    check_result $?
+  else
+    echo "[SKIP] Multistage build from Dockerfile - $FULL_IMAGE_NAME does not exists."
+  fi
+
```

Comment:
Commit title: If full image does not exists, skip the multistage build test

My question is: If the full image does not exist, then there's likely to be some problem with the way we detect the FULL_IMAGE_NAME. Or why do you suppose the full image would not exist?

Reply:
For example, right now, we are preparing the minimal Python container image based on C9S (per request) but the full image for the same combination of Python version and platform does not exist. I see this as a possible future if the minimal images will be popular, we might start producing them more and more without their full counterparts.

Reply:
However, if we don't have the full counterparts, the multistage build feature won't be possible, and they might decrease in popularity.

Reply:
Discussed in person: Let's try to release just the minimal for c9s and see the reactions.

Reply:
Note that c9s (and c8s) full sized images are not planned to be created for Python, as the UBI images should cover most of the use-cases. So if the multistage build is important for the Python images, we should likely make it work with the fullsized ubi images in future.

Reply:
Sounds good. However, I don't consider this a blocker for this PR. We can implement it later if/when we get some feedback wanting multistage builds.